docs/configuration.rst

```diff
@@ -79,6 +79,10 @@ You may want to pull/push images from/to a custom docker registry. For instance,
 Open edX customisation
 ~~~~~~~~~~~~~~~~~~~~~~
 
+- ``OPENEDX_PLATFORM_REPOSITORY`` (default: ``"https://github.com/openedx/edx-platform.git"``)
+
+This defines the git repository from which you install Open edX platform code. If you run an Open edX fork with custom patches, set this to your own git repository.
```

Comment:
Can you you please highlight the fact that the setting will be overridden by the `--build-arg` argument? In reality we can say that this new setting does not completely define the edx-platform repo but just a reasonable default.

Reply:
We already say that under the subheading _openedx Docker Image build arguments,_ don't we?